staging/src/k8s.io/client-go/tools/cache/store.go

```diff
@@ -199,17 +199,17 @@ func (c *cache) Index(indexName string, obj interface{}) ([]interface{}, error)
 	return c.cacheStorage.Index(indexName, obj)
 }
 
-func (c *cache) IndexKeys(indexName, indexKey string) ([]string, error) {
```

Comment:
add `IndexKeys returns the storage keys of the stored objects whose set of indexed values for the named index includes the given indexed value. The returned keys are suitable to pass to GetByKey().` godoc here, this signature is really hard to understand otherwise